data.sql

```diff
@@ -3,75 +3,104 @@ INSERT INTO animals (
         date_of_birth,
         escape_attempts,
         neutered,
-        weight_kg species
+        weight_kg species,
+        species_id,
+        owner_id
     )
 VALUES (
         'Agumon',
         '2020-02-03',
         0,
         TRUE,
-        10.23
+        10.23,
+        2,
+        1
     ),
     (
         'Gabumon',
         '2018-11-15',
         2,
         TRUE,
-        8
+        8,
+        2,
+        2
     ),
     (
         'Pikachu',
         '2021-01-07',
         1,
         FALSE,
-        15.04
+        15.04,
+        1,
+        2
     ),
     (
         'Devimon',
         '2017-05-12',
         5,
         TRUE,
-        11
+        11,
+        2,
+        3
     ),
     (
         'Charmander',
         '2020-02-08',
         0,
         FALSE,
-        -11
+        11,
+        1,
+        4
     ),
     (
         'Plantmon',
         '2021-11-15',
         2,
         TRUE,
-        -5.7
+        5.7,
+        2,
+        3
     ),
     (
         'Squirtle',
         '1993-04-02',
         3,
         FALSE,
-        -12.13
+        12.13,
+        1,
+        4
     ),
     (
         'Angemon',
         '2005-04-2',
         1,
         TRUE,
-        -45
+        45,
+        2,
+        5
     ),
     (
         'Boarmon',
         '2005-06-07',
         7,
         TRUE,
-        20.4
+        20.4,
+        2,
+        5
     ),
     (
         'Ditto',
         '2022-05-14',
         4,
         TRUE,
         22
+    ),
+    (
+        'Blossom',
+        '1998-10-13',
+        3,
+        TRUE,
+        17,
+        1,
+        2
     );
```

Comment:
- [ ] You're missing some major insertions in the Data.sql file. Below you'll find a screenshot of them all.

### Instructions:
![image](https://github.com/devalibello/vet-clinic/assets/50990925/1c04e3d4-0370-478a-893b-70632a2cbca2)